x/group/spec/01_concepts.md

```diff
@@ -1,70 +0,0 @@
-# Concepts
```

Comment:
We should also keep the specification, which is still available to view on the documentation site although not front and center with its own section under modules. The link to the group module under the experimental build still works and we should keep these specs for as long as the experimental build includes the group module.